README.md

```diff
@@ -48,6 +50,18 @@ Content:
    You should now have a working kubernetes cluster.
    You should also have an encrypted kubeconfig at `<CK8S_CONFIG_PATH>/.state/kube_config_<prefix>.yaml` that you can use to access the cluster.
 
+## Changing authorized SSH keys for a cluster
+
+Authorized SSH keys can be changed for a cluster using:
+
+```bash
+./bin/ck8s-kubespray apply-ssh <prefix>
+```
+
+It will set the public SSH key(s) found in`<CK8S_CONFIG_PATH>/<prefix>-config/ssh-keys.pub` as authorized keys in your cluster (to apply multiple keys, put one key per line in the file). Note that the authorized SSH keys for the cluster will be set to these keys _exclusively_, removing any keys that may already be authorized, so make sure the file includes **every SSH key** that should be authorized.
```

Comment:
I think it would make more sense to call the file `authorized_hosts`. This is how it is "usually done" with ssh and it goes well together with the rest of the text. :slightly_smiling_face: 
```suggestion
It will set the public SSH key(s) found in`<CK8S_CONFIG_PATH>/<prefix>-config/authorized_hosts` as authorized keys in your cluster (to apply multiple keys, put one key per line in the file). Note that the authorized SSH keys for the cluster will be set to these keys _exclusively_, removing any keys that may already be authorized, so make sure the file includes **every SSH key** that should be authorized.
```